source/_integrations/homekit_controller.markdown

```diff
@@ -45,67 +45,79 @@ ha_integration_type: integration
 
 The [HomeKit](https://developer.apple.com/apple-home/) controller integration allows you to connect accessories with the "Works with HomeKit" logo to Home Assistant. This integration should not be confused with the [HomeKit](/integrations/homekit/) integration, which allows you to control Home Assistant devices via HomeKit.
 
-The integration will automatically detect HomeKit compatible devices that are ready to pair if the [`zeroconf`](/integrations/zeroconf/) integration is enabled. This is enabled by default on new installations via the [`default_config`](/integrations/default_config/) integration.
+# Adding a HomeKit device
 
-To see which devices have been discovered see the "Integrations" page in your Home Assistant dashboard. When you click on "Configure" you can enter your HomeKit PIN and the device should be added to your Home Assistant instance. If your device is currently paired with an Apple device via HomeKit, you will need to reset it in order to pair it with Home Assistant. Once Home Assistant is configured to work with the device, you can export it back to Siri and Apple Home with the [`HomeKit`](/integrations/homekit/) integration.
+There are different methods to add a HomeKit device to Home Assistant:
 
-## Supported devices
-
-There is currently support for the following device types within Home Assistant:
+- [via Ethernet or Wi-Fi](#adding-a-homekit-device-via-ethernet-or-wi-fi)
+- [via Bluetooth](#adding-a-homekit-device-through-bluetooth)
+  - [migrate to a Thread network via Home Assistant](#adding-a-homekit-device-to-a-thread-network-via-home-assistant)
+  - [migrate to Thread network via Apple Thread border router](#adding-a-homekit-device-via-apple-thread-border-router)
 
-- Alarm Control Panel (HomeKit security system)
-- Climate (HomeKit thermostats and air conditioners)
-- Cover (HomeKit garage door openers, windows, or window coverings)
-- Light (HomeKit lights)
-- Lock (HomeKit lock)
-- Switch (HomeKit switches, outlets and valves)
-- Binary Sensor (HomeKit motion, contact, occupancy, carbon monoxide and smoke sensors)
-- Sensor (HomeKit humidity, temperature, co2 and light level sensors)
-- Fan
-- Air Quality
-- Humidifier (HomeKit humidifiers and dehumidifiers)
-- Automation Triggers (HomeKit 'stateless' accessories like buttons, remotes and doorbells)
+## Adding a HomeKit device via Ethernet or Wi-Fi
 
-<div class='note'>
+The HomeKit controller integration automatically detects HomeKit [compatible devices](#supported-devices) that are ready to pair if the [`zeroconf`](/integrations/zeroconf/) integration is enabled. The `zeroconf` is enabled by default on new installations via the [`default_config`](/integrations/default_config/) integration.
```

Comment:
Since we have this in the prerequisites now, and most people have default_config, I would suggest to simplify this section.

```suggestion
The HomeKit controller integration automatically detects HomeKit [compatible devices](#supported-devices) on your network when they are ready to pair.
```